src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const listItems = [...document.querySelectorAll('li')];
+
+function convertToNumber(string) {
+  return Number(string.split('$').join('').replaceAll(',', ''));
+}
+
+function sortList(list) {
+  list.sort((a, b) => {
+    return convertToNumber(b.dataset.salary)
+    - convertToNumber(a.dataset.salary);
+  });
+
+  document.querySelector('ul').append(...list);
+};
+
+sortList(listItems);
+
+function getEmployees(list) {
+  list.map(item => {
+    return {
```

Comment:
And here you can use implicit return too.

It's not a mistake. It's just code optimization